jekyll/_cci2/server/installation/phase-3-execution-environments.adoc

```diff
@@ -0,0 +1,949 @@
+      encryption:
+        key: "" # "" indicates the secret nomad-gossip-encryption-key already exists
+----
+
+* *Option 2 - CircleCI creates the secret*
++
+Add the value to `values.yaml`. CircleCI will create the secret automatically.
+
++
+[source,yaml]
+----
+nomad:
+  server:
+    gossip:
+      encryption:
+        key: <secret-key-32-chars>
+----
+
+[#nomad-mtls]
+=== Nomad mTLS
+
+The `CACertificate`, `certificate` and `privateKey` can be found in the output of the terraform module.  They must be base64 encoded.
+
+[source,yaml]
+----
+nomad:
+  server:
+    ...
+    rpc:
+      mTLS:
+        enabled: true
+        certificate: <base64-encoded-certificate>
+        privateKey: <base64-encoded-private-key>
+        CACertificate: <base64-encoded-ca-certificate>
+----
+
+[#nomad-autoscaler]
+=== Nomad Autoscaler
+
+If you have enabled the nomad autoscaler also include the following section under `nomad`.
+
+[source,yaml]
+----
+nomad:
+  ...
+  auto_scaler:
+    enabled: true
+    scaling:
+      max: <max-node-limit>
+      min: <min-node-limit>
+----
+
+[#aws]
+=== AWS
+the values for this section were created <<nomad-autoscaler-aws,here>>
+
+[source,yaml]
+----
+nomad:
+  ...
+  auto_scaler:
+    enabled: true
+    scaling:
+      max: <max>
+      min: <min>
+
+    aws:
+      enabled: true
+      region: <region>
+      autoScalingGroup: <asg-name>
+
+      accessKey: <access-key>
+      secretKey: <secret-key>
+      # or
+      irsaRole: <role-arn>
+----
+
+[#gcp]
+=== GCP
+the values for this section were created <<nomad-autoscaler-gcp,here>>
+
+[source,yaml]
+----
+    gcp:
+      enabled: true
+      project_id: <project-id>
+      mig_name: <instance-group-name>
+
+      region: <region>
+      # or
+      zone: <zone>
+
+      workloadIdentity: <service-account-email>
+      # or 
+      service_account: <service-account-json>
+----
+
+[#nomad-clients-validation]
+=== Nomad Clients Validation
+
+CircleCI has created a project called https://github.com/circleci/realitycheck[realitycheck] which allows you to test your Server installation. We are going to follow the project so we can verify that the system is working as expected. As you continue through the next phase, sections of realitycheck will move from red to green.
+
+To run realitycheck, you need to clone the repository. Depending on your GitHub setup, you can use one of the following commands:
+
+[#github-cloud]
+==== GitHub Cloud
+
+[source,shell]
+----
+git clone https://github.com/circleci/realitycheck.git
+----
+
+[#github-enterprise]
+==== GitHub Enterprise
+
+[source,shell]
+----
+git clone https://github.com/circleci/realitycheck.git
+git remote set-url origin <YOUR_GH_REPO_URL>
+git push
+----
+
+Once you have successfully cloned the repository, you can follow it from within your CircleCI server installation. You need to set the following variables. For full instructions please see the https://github.com/circleci/realitycheck/tree/server-3.0[repository readme].
```

Comment:
These env vars etc aren't mentioned in the readme I don't think? https://github.com/circleci/realitycheck#prerequisites-1 @circleci/server